### PR TITLE
Issue 662: Minor fix in documentation

### PR DESCRIPTION
**Pull request by shubham90**
Minor fix in the documentation 
https://pulp.plan.io/issues/662
fixes #662

### CODE REVIEW

**Comment by mhrivnak** [comment]
ok test

**Comment by ipanova** [comment]
@shubham90 Since you made changes in platform, this will affect the plugins also, so you would need to update them.
```
[ipanova@ina pulp_development]$ cd pulp_rpm/
[ipanova@ina pulp_rpm]$ git grep "remove cop"
extensions_admin/pulp_rpm/extensions/admin/structure.py:DESC_REMOVE = _('remove copied or uploaded modules from a repository')
[ipanova@ina pulp_rpm]$ cd ..
[ipanova@ina pulp_development]$ cd pulp_puppet/
[ipanova@ina pulp_puppet]$ git grep "remove cop"
pulp_puppet_extensions_admin/pulp_puppet/extensions/admin/repo/remove.py:DESC_REMOVE = _('remove copied or uploaded modules from a repository')
```

**Comment by ipanova** [comment]
also, do not forget to update the state of the issue https://pulp.plan.io/issues/662

**Comment by shubham90** [comment]
Thanks for the comments @ipanova . I had already submitted a pull request in pulp_rpm plugin  for this issue. Will do the same for pulp_puppet.

**Comment by rbarlow** [comment]
ok test

File: docs/user-guide/admin-client/tasks.rst [hunk 1]
```diff
@@ -71,7 +71,7 @@ in the list.
 The ``reaper`` task is responsible for cleaning up the database on a regularly scheduled interval.
 The interval is configured with ``reaper_interval`` in ``[data_reaping]`` section of
 ``/etc/pulp/server.conf``. The value can be whole or fraction of days. The length of time to keep
-documents for each collection is also configured in the same section. ``archived_calls``,
+documents for each collection is also configured in the same section.
```
Comment (rbarlow) [inline docs/user-guide/admin-client/tasks.rst]: Why did you remove ``archived_calls`` here?

File: nodes/test/nodes_tests/base.py [hunk 2]
```diff
@@ -49,9 +48,6 @@ def setUpClass(cls):
         shutil.rmtree(storage_dir+'/*', ignore_errors=True)
         managers.initialize()
 
-    def setUp(self):
-        QueuedCall.get_collection().remove()
```
Comment (rbarlow) [inline nodes/test/nodes_tests/base.py]: Why is this being removed?

File: server/etc/pulp/server.conf [hunk 3]
```diff
@@ -164,7 +163,6 @@
 
 [data_reaping]
 # reaper_interval: 0.25
-# archived_calls: 0.5
```
Comment (rbarlow) [inline server/etc/pulp/server.conf]: And these changes?

**Comment by rbarlow** [comment]
@shubham90 There are quite a few changes in here that do not seem to be related to issue 662. Perhaps it is because you are working out of your master branch?

**Comment by shubham90** [comment]
@rbarlow yeah! I took the issue to a new branch but it seems that I already added these changes to master. Will keep this in mind from next time.
